docs/menu.yml

```
@@ -203,7 +203,11 @@ catalog:
           - name: "Kubernetes"
             catalog:
               - name: "Observe Kubernetes"
-                path: "/en/setup/backend/backend-k8s-monitoring"
+                catalog:
+                  - name: "kube-state-metrics + cAdvisor"
+                    path: "/en/setup/backend/backend-k8s-monitoring-metrics-cadvisor"
+                  - name: "Rover"
+                    path: "/en/setup/backend/backend-k8s-monitoring-rover"
```

```suggestion
                  - name: "Observe Kubernetes"
                    path: "/en/setup/backend/backend-k8s-monitoring"
                  - name: "kube-state-metrics + cAdvisor"
                    path: "/en/setup/backend/backend-k8s-monitoring-metrics-cadvisor"
                  - name: "Rover"
                    path: "/en/setup/backend/backend-k8s-monitoring-rover"
```

Please setup the docs in this way.